packages/core/src/modules/credentials/__tests__/V2CredentialService.cred.test.ts

```diff
@@ -806,11 +806,11 @@ describe('CredentialService', () => {
 
       const credential = mockCredentialRecord()
       mockFunction(credentialRepository.getById).mockReturnValue(Promise.resolve(credential))
-
+      credential.credentials[0].credentialRecordId
       await credentialService.deleteById(credential.id, {
         deleteAssociatedCredentials: true,
       })
-      expect(storeCredentialMock).toHaveBeenNthCalledWith(1, credential.credentials[0].credentialRecordId)
+      expect(storeCredentialMock).toHaveBeenNthCalledWith(1, credential.id)
```

Comment:
also, it should probably be called deleteCredentialMock?

```suggestion
      expect(deleteCredentialMock).toHaveBeenNthCalledWith(1, credential.id)
```